### ISSUE TITLE
Issue w/ opera

### ISSUE THREAD

**Issue by oham9** [altdesktop/playerctl#333]
I'm trying to use on Opera (111.0.5168.25), but it doesn't appear upon using `playerctl -l`.
(spotify does appear and work just fine).

Am I forgetting to do something, or should I install something on opera to make it work? Please let me know. 

Specs: 
```
Ubuntu    == 22.04
playerctl == v2.4.1
Opera     == 111.0.5168.25
```

**Comment by keithbowes** [comment]
Yeah, I installed playerctl due to the requirement for waybar and it seems that most things aren't working. Firefox isn't, Chromium isn't, mpv isn't. VLC is, though.

UPDATE: I installed the mpv-mpris plugin and playerctl can control mpv fine now. Chromium works as long as you're watching a video (but I get a "no player" message when not watching a video). Same in Firefox, after I reenabled hardware media keys (I previously disabled it to stop Firefox from creating thumbnails of every video I watch). Chromium's setting can be accessed via chrome://flags/#hardware-media-key-handling; maybe Opera has it disabled by default.
